frontend/src/components/Team.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import {
+  Card, CardImg, CardTitle, CardText,
+  CardSubtitle, CardBody, Row, Col
+} from 'reactstrap';
+import '../styles/Team.css';
+import data from '../jsonData/TeamData.json'
+
+const cardStyle = {
+  backgroundColor: '#E4A58F',
+  borderWidth: 'thick',
+  borderColor: '#333',
+}
+
+function Team() {
+  return (
+    <div id="team">
+      <Row>
+        {data.map((person, index) => (
+          <Col sm="4">
+            <div key={index}>
+              <Card body inverse style={cardStyle}>
+                <CardImg top width="100%" src={person.image} alt="Card image cap" />
```

Comment:
The cards for Abagail and Renee are shorter in height than the rest of the cards. You can tell by the margin difference between them which doesn't exist for the others. You can remove this difference by setting a predetermined height for the CardImg and also add margin space between the cards so that the borders of the cards are not right next to each other.

Reply:
I helped make some changes to the css @roseyume let me know what you think and if it needs to change.

Reply:
Looks good